conf_linux_public.py

```diff
@@ -88,16 +88,18 @@ def get_building_cmd(command, gcc_latest, enable_devtoolset):
     else:
         return f'{enable_devtoolset} && {command}' #enable new compiler on CentOS
 
-
+#Repositories which will be downloaded
 PRODUCT_REPOS = [
     {'name': 'MediaSDK'},
+    {'name': 'libva'},
```

Comment:
clone libva with using the code which will be written here (because need to use manifest) or in opposite extend API